### PR TITLE
FR fix issues #128 & #131 

### PR DESCRIPTION
**Pull request by adelourme**
Hello, we changed and corrected formats and values of certificate, dates and RAT test name and manufacturer.
Thanks

### CODE REVIEW

File: FR/2DCode/raw/recovery_ok.json [hunk 1]
```diff
@@ -21,15 +21,15 @@
     }
```
Comment (bhavin-qryptal) [inline FR/2DCode/raw/recovery_ok.json:21]: Value of ci should be dgci:V1:FR:EEVRA31SARJ3X5H0IT3V1IFES:31

File: FR/2DCode/raw/vaccin_ok.json [hunk 2]
```diff
@@ -24,16 +24,16 @@
     }
```
Comment (bhavin-qryptal) [inline FR/2DCode/raw/vaccin_ok.json:24]: Value of ci should be dgci:V1:FR:AP6D4CPJVI1RHAHF6OBPQM0CG:84

File: FR/2DCode/raw/test_pcr_ok.json [hunk 3]
```diff
@@ -25,16 +25,16 @@
     }
```
Comment (bhavin-qryptal) [inline FR/2DCode/raw/test_pcr_ok.json:25]: Value of ci should be dgci:V1:FR:QSMFF6WU45QA0F0XTD2HS8WPO:10

File: FR/2DCode/raw/test_pcr_ok.json [hunk 4]
```diff
@@ -25,16 +25,16 @@
...
- "COSE": "D2844DA20448D6FC694CB81CEB0B0126A0590116A4041A60F2D565061A60A3BB6501624652390103A101A4617481AB6273631B000001797F8BDF18626D616C3344204D65646963696E65736264721B000001797F8BDF28627474684C50363436342D34626E6D71323031392D6E436F562052542D715043526274636E54657374696E672043656E74726562636F6246526263697827646763693A56313A46523A455530535142583651474E493450373052324449454B3442593A31306269736A4C61626F7261746F72796274676938343035333930303662747269323630343135303030636E616DA463666E74664455504F4E4462666E664475706F6E6463676E74654D4152494562676E654D617269656376657265312E302E3063646F626A313936322D30372D30315840FD1A9BC486592856D01500D4D7366B6D5131F8CF776BAD7C828AC0263BF9D1AE8C226003FDE396AAB52C40E453C3F31421E6D1FD8B343261CCA925E5E6A50A26",
```
Comment (bhavin-qryptal) [inline FR/2DCode/raw/test_pcr_ok.json:29]: This COSE payload is not compliant with expected JSON Schema.
Value of dr and sc should not be int but date value in ISO 8601 format.

image

**Review by bhavin-qryptal — COMMENTED** [review]
It would be ideal if a couple of more issues are fixed. Refer to inline comments for details.

**Comment by adelourme** [comment]
All issues fixed please could you commit changes
Thanks

File: FR/2DCode/raw/test_pcr_ok.json (hunk 3 above)
Comment (adelourme) [reply FR/2DCode/raw/test_pcr_ok.json:25]: fix ok

File: FR/2DCode/raw/test_pcr_ok.json (hunk 4 above)
Comment (adelourme) [reply FR/2DCode/raw/test_pcr_ok.json:29]: fix ok

File: FR/2DCode/raw/vaccin_ok.json (hunk 2 above)
Comment (adelourme) [reply FR/2DCode/raw/vaccin_ok.json:24]: fix ok

File: FR/2DCode/raw/recovery_ok.json (hunk 1 above)
Comment (adelourme) [reply FR/2DCode/raw/recovery_ok.json:21]: fix ok

Comment (bhavin-qryptal) [reply FR/2DCode/raw/recovery_ok.json:21]: Verified OK

File: FR/2DCode/raw/test_pcr_ok.json (hunk 3 above)
Comment (bhavin-qryptal) [reply FR/2DCode/raw/test_pcr_ok.json:25]: 1. Value of ci in JSON does not match with CBOR / COSE payload.
2. Value of 'dr' and 'sc' under 't' are still int. They should be date value in ISO format.

image

**Review by SchulzeStTSI — APPROVED** [review]

**Comment by adelourme** [comment]
You can create a new issue please.
We are going to fix issues needs to be fixed quickly.
Thanks

**Pull request merged**